delta/plugins/graph-analytics/src/test/resources/indexed-documents-after-query.json

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
Quite a big chunk but it is meant to be sure that the painless script does not have undesired side effects